content/zpages.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Z-Pages"
+type = "leftnav"
++++
+
+OpenCensus provides in-process web pages that displays
+collected data from the process. These pages are called z-pages
+and they are useful to see collected data from a specific process
+without having to depend on any metric collection or
+distributed tracing backend.
+
+Z-Pages can be useful during the development time or when
+the process to be inspected is known in production.
+Z-Pages can also be used to debug [exporter](/exporters) issues.
+
+In order to serve Z-pages, register their handlers and
+start a web server. Below, there is an example how to
+serve these pages from `127.0.0.1:7777/debug`.
+
+
+{{% snippets %}}
+{{% go %}}
+```
+import "go.opencensus.io/zpages"
+
+zpages.Handle(nil, "/debug")
+log.Fatal(http.ListenAndServe("127.0.0.1:7777", nil))
+```
+{{% /go %}}
+{{% java %}}
+```
+// Add the dependencies by following the instructions at
+// https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/zpages
+
+ZPageHandlers.startHttpServerAndRegisterAll(7777);
+```
+{{% /java %}}
+{{% /snippets %}}
+
+Once handler is registered, there are various pages provided
+from the libraries:
+
+* [127.0.0.1:7777/debug/rpcz](http://127.0.0.1:7777/debug/rpcz)
+* [127.0.0.1:7777/debug/tracez](http://127.0.0.1:7777/debug/tracez)
```

Comment:
Shall we mention [statsz](https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/zpages#view-measures-and-stats-for-all-exported-views-on-statsz-page) and [traceconfigz](https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/zpages#view-and-update-tracing-configuration-on-traceconfigz-page) here?

Reply:
I want to as soon as Go has support for these.

Reply:
+1.